components/atoms/CardLink/CardLinkStyles.ts

```diff
@@ -7,7 +7,7 @@ interface Props {
 export const StyledCardLink = styled.a<Props>`
     cursor: pointer;
     height: 100%;
-    max-height: 377px;
+    max-height: 450px;
```

Comment:
This seems restrictive and may cause problems when content is stacked or flows naturally with responsiveness. I'm seeing issues on and below Safari 15.1 on Monterey.

The card margins are ignored and the link element wrapping the card has `margin-top: 0;`.

Could we add `max-height: 100%;` instead to the card? It seems to fix the issue.

![bs_macmty_Safari_15 1](https://user-images.githubusercontent.com/1733936/148110358-79a5e128-b058-49cc-b977-4d3479d37d39.jpg)

Reply:
Sounds good to me!